lang/Jailbreak.English/Generic/GenericCommandNotifications.cs

```diff
@@ -10,7 +9,7 @@ namespace Jailbreak.English.Generic;
 
 public class GenericCommandNotifications : IGenericCommandNotifications, ILanguage<Formatting.Languages.English>
 {
-	public static FormatObject PREFIX =
+	public static FormatObject GENERIC_PREFIX =
```

Comment:
Renamed all prefix's so I can statically include them in my Notification implementation